lib/.eslintrc.yaml

```diff
@@ -23,8 +23,6 @@ rules:
       message: Use an error exported by the internal/errors module.
     - selector: CallExpression[callee.object.name='Error'][callee.property.name='captureStackTrace']
       message: Please use `require('internal/errors').hideStackFrames()` instead.
-    - selector: AssignmentExpression:matches([left.name='prepareStackTrace'], [left.property.name='prepareStackTrace'])
```

Comment:
instead of removing this, you can add `// eslint-disable-next-line no-restricted-syntax`